houghtool/__init__.py

```diff
@@ -609,25 +615,30 @@ def intersection(line1, line2, tolerance=1e-6, subpixel=False):
 
 
 def draw_lines(img, lines, color=None, thickness=1):
+    num_lines_drawn = 0
     h, w = img.shape[:2]
     color = color or (0, 255, 0) if len(img.shape) == 3 else 255
     _linetype = linetype(lines[0])
     if _linetype == RHOTHETA:
         lines = [convert(line, bbox=[0, 0, w, h]) for line in lines]
     for line in lines:
         for x1, y1, x2, y2 in line:
-            if ERRORVAL in [x1, y1, x2, y2]:
-                print("Line {} has np.nan vals; not drawing.".format(line))
+            # if ERRORVAL in [x1, y1, x2, y2]:
+            if True in np.isnan([x1, y1, x2, y2]):
+                # print("Line {} has np.nan vals; not drawing.".format(line))
```

Comment:
I should use a logger instead of these print statements...

Reply:
Its not something really crucial, I don't mind if you want to keep it unchanged for now.